test/ecto/query/builder/join_test.exs

```diff
@@ -137,20 +137,6 @@ defmodule Ecto.Query.Builder.JoinTest do
     end
   end
 
-  test "raises on non-atom as" do
```

Comment:
I tried just changing this test to use a string binding, but got an error about a malformed join. But it works in my manual tests so I'm not sure whats up. Didn't want to invest too much time into it w/o knowing if I'm moving in the generally correct direction.